cmd/tools/vdoc/theme/dark-mode.js

```diff
@@ -1,6 +1,5 @@
-(function() {
```

Comment:
You can also use arrow functions if you upgrade to the newest browsers.

Reply:
The two iffes in the code were the parts I was most unsure about. Afaik it works without the function wrapper so I assumed they just encapsulated the initialization code in a block  - for a readability reason?

I had this changed for a moment to an arrorw function, but as they are on the same level as  the function declarations I left the `function` keyword. To extend my elaboration a little bit: for normal declarations I came to prefer `function myFn() {` to `const myFn = () => {`  - after using the latter exclusively for a while. It's more immediately visible while reading that it's a function not a variable I think. Also the main way functions are declared i.e. in the Svelte framework.